internal/devicecfg/devicecfg.go

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) 2021 Nordix Foundation.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package devicecfg provides service domain to device config
+package devicecfg
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/networkservicemesh/sdk-sriov/pkg/tools/yamlhelper"
+	"github.com/networkservicemesh/sdk/pkg/tools/log/logruslogger"
+)
+
+// Config contains list of available service domains
+type Config struct {
+	Interfaces []*Resource `yaml:"interfaces"`
```

Comment:
Do you need interfaces for ovs also?

Reply:
@ljkiraly yes, the l2 connection point is either existing physical interface or existing ovs bridge, when interface name is provided, then ovs forwarder creates a normal ovs bridge (the latest 'config' file takes bridge name as input parameter under 'interface') and adds that interface as ovs port into the bridge.